src/vs/workbench/common/contextkeys.ts

```diff
@@ -70,6 +70,7 @@ export const IsCenteredLayoutContext = new RawContextKey<boolean>('isCenteredLay
 export const SplitEditorsVertically = new RawContextKey<boolean>('splitEditorsVertically', false, localize('splitEditorsVertically', "Whether editors split vertically"));
 export const EditorAreaVisibleContext = new RawContextKey<boolean>('editorAreaVisible', true, localize('editorAreaVisible', "Whether the editor area is visible"));
 export const EditorTabsVisibleContext = new RawContextKey<boolean>('editorTabsVisible', true, localize('editorTabsVisible', "Whether editor tabs are visible"));
+export const inDiffEditor = new RawContextKey<boolean>('inDiffEditor', false, localize('inDiffEditor', "Whether the context is a diff editor"));
```

Comment:
But there already is `EditorContextKeys.inDiffEditor`?

Reply:
I now fetch InDiffEditor from `EditorContextKeys`. Previously I wasn't aware I could get it from here so imported it but this worked thanks.